### ISSUE TITLE
`getOwnedUTXOs().length` assert(0)

### ISSUE THREAD

**Issue by kchulj** [bosagora/faucet#96]
The faucet is unable to retrieve its owned `UTXO`s. This error prevents the faucet from sending additional transactions.

```
core.exception.AssertError@source/faucet/main.d(176): Assertion failure
----------------
??:? _d_assert [0x105071407]
??:? @safe bool faucet.main.State.update(agora.api.FullNode.API, agora.common.Types.Height) [0x104f37cba]
??:? void faucet.main.Faucet.send() [0x104f38623]
??:? void faucet.main.main(immutable(char)[][]).__lambda21() [0x104f3a988]
??:? nothrow @safe void vibe.core.core.setTimer(core.time.Duration, void delegate(), bool).__lambda4() [0x104c2ef43]
??:? nothrow @safe void vibe.core.core.createTimer(void delegate() nothrow @safe).C.opCall(vibe.core.core.Timer).__lambda2(vibe.core.core.Timer) [0x104c2f281]
??:? nothrow void vibe.core.task.TaskFuncInfo.set!(void delegate(vibe.core.core.Timer) nothrow @safe, vibe.core.core.Timer).set(ref void delegate(vibe.core.core.Timer) nothrow @safe, ref vibe.core.core.Timer).callDelegate(ref vibe.core.task.TaskFuncInfo) [0x104c6ece5]
??:? void vibe.core.task.TaskFuncInfo.call() [0x104c6c34b]
??:? nothrow void vibe.core.task.TaskFiber.run() [0x104c6b898]
??:? fiber_entryPoint [0x10508ea23]
```

**Comment by Geod24** [comment]
Probably shouldn't be an `assert` since it can happen. Do you have genesis in the list of keys ?

**Comment by kchulj** [comment]
> Probably shouldn't be an `assert` since it can happen. Do you have genesis in the list of keys ?

Yes, `boa1xzgenes5cf8xel37fz79gzs49v56znllk7jw7qscjwl5p6a9zxk8zaygm67:**SECRET**` is there.

**Comment by kchulj** [comment]
After the 1st block, `.filter!(tup => tup.value.output.address in secret_keys)` returns an empty AA.

**Comment by kchulj** [comment]
Fixed by #97
